llvm/test/tools/llvm-lib/arm64ec-implib.test

```diff
@@ -96,6 +96,11 @@ READOBJ-NEXT: Name type: name
 READOBJ-NEXT: Export name: dataexp
 READOBJ-NEXT: Symbol: __imp_dataexp
 
+Using -machine:arm64x gives the same output.
+RUN: llvm-lib -machine:arm64x -def:test.def -out:test.lib
```

Comment:
Use a different name for this lib file, for debuggability?

Reply:
I changed it in the new version (it required changing check patterns to accept different names).